### ISSUE TITLE
发送和接收功能优化

### ISSUE THREAD

**Issue by US00126** [localsend/localsend#1269]
### Is your feature request related to a problem? Please describe.

发送功能中没有历史记录查询，如果本地使用者想要查询本地用户发送的历史记录是无法查询的。但是接受方时候接受的历史记录的。
![发送记录优化建议](https://github.com/localsend/localsend/assets/118803752/b0ed38e8-66ec-4590-889d-93ef47407507)

### Describe the solution you'd like

发送是否可以增加一个发送历史记录查询功能，和接收历史记录对应，便于使用者查询，最好去掉可以去掉随意删除历史记录的功能，以便后续追踪。

**Comment by OpenJarvisAI** [comment]
没看懂，发送和接收不都是你自己吗？

**Comment by Caesarovich** [comment]
It would be nice if the issue was written in English for the whole community to be able to understand it.

**Comment by Jacen918** [comment]
That's right, in fact, I prefer that it be directly made into a conversational chat window, which is more intuitive and convenient.
